src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,78 @@
+package duke;
+
+import duke.exceptions.DukeException;
+import duke.data.LocalStorage;
+import duke.handlers.*;
+import duke.models.*;
+import duke.services.Parser;
+import duke.utils.Commands;
+
+import java.util.Scanner;
+import static duke.services.Ui.dukePrint;
+
+public class Duke {
+
+    private final LocalStorage storage;
+    private final Parser parser;
+    private TaskList list;
```

Comment:
Perhaps the TaskList attribute could have a better name such as tasks. In the coding standards, plural form should be used on names representing a collection of objects, maybe you could look at renaming it to a plural form.